### ISSUE TITLE
Deserialisation issues with streaming events (specifically favourites)

### ISSUE THREAD

**Issue by vikingcode** [nickhodge/BoxKite.Twitter#27]
`MapFromStreamTo<T>` throws 'Newtonsoft.Json.JsonSerializationException' as its trying to deserialise a string as a bool. 

Investigating, will send PR if I find the solution

**Comment by vikingcode** [comment]
Right, normally the JSON that'd be deserialised to `BoxKite.Twitter.Models.User` doesn't even have a source. The property could be dropped entirely or just modified to `string`. I'm not sure what that property serves on the user obj (from twitters point of view).

Thoughts @nickhodge?

**Comment by nickhodge** [comment]
This must be in streaming when an event of a particular type flows in (I think)

Will re-investigate
